documentation/src/main/asciidoc/user_guide/server_protocols_hotrod.adoc

```diff
@@ -83,6 +84,284 @@ which means that the value should be taken from a system property named jboss.bi
 
 For a complete reference of the available configuration options for the properties file please refer to link:{javadocroot}/org/infinispan/client/hotrod/RemoteCacheManager.html[RemoteCacheManager]'s javadoc.
 
+==== Authentication
+
+If the server has set up authentication, you need to configure your client accordingly. Depending on the mechs enabled on the server,
+the client must provide the required information.
+
+NOTE: This section is about client configuration. If you want to set up the server to require authentication, read the
+link:link:../server_guide/server_guide.html#security:hotrod_auth[Hot Rod server authentication] section.
+
+===== DIGEST-MD5
+DIGEST-MD5 is the recommended approach for simple username/password authentication scenarios. If you are using the default
```

Comment:
IMO we should document that MD5 should not be used, and also we should not use it as a default algorithm.